app/data_models/answer_store.py

```diff
@@ -137,15 +137,17 @@ def remove_answer(
 
         return False
 
-    def remove_all_answers_for_list_item_id(self, list_item_id: str) -> None:
-        """Remove all answers associated with a particular list_item_id.
+    def remove_all_answers_for_list_item_ids(
+        self, list_item_ids: Iterable[str]
+    ) -> None:
```

Comment:
I suggest instead of making this an iterable that is sometimes called with a single item of a list, use variable length args, i.e `*list_item_ids`? The is can be called with 1 or more list item ids.

Reply:
Yep - I like that, done 👍